proposals/extensions.md

```diff
@@ -995,6 +1015,25 @@ extension member lookup will find the `int` property and stop there.
 
 For context see [extension method invocation rules](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#11783-extension-method-invocations).
 
+### More specific extension member
```

Comment:
>More specific extension member [](http://example.com/codeflow?start=4&length=30)

Should this section also define a concept of "less specific type than `X`"?

Reply:
I reframed this section to define "less specific" to avoid confusion

Reply:
This thread was marked as resolved, but I do not see it addressed. The question was about "specific **type**" vs. "specific member".